### DISCUSSION TITLE
instruments not resetting on new playback

### DISCUSSION THREAD

**Discussion by DeWolfRobin** [mido/mido#441] [Q&A]
I have a small program that plays midi files on loop. When one song ends, usually the instruments reset, but sometimes the instruments caries over and the song plays in trumpet (or whatever the previous instrument was) instead of piano (or whatever the new instrument is). I tried outport.panic() and outport.reset(), but those don't seem to solve the issue. Is there any way to properly reset the midi controller?

**Comment by rdoursenaud** [comment]
If `reset()` does not work, it likely means that either your equipment is ignoring it.
What MIDI device do you use for playback?
